packages/jest-matchers/src/utils.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+// Stringify values and remove circular references
+function stringify(obj: any): string {
+  const set = new Set();
+  return JSON.stringify(obj, (key, value) => {
+    if (typeof value === 'object' && value !== null) {
+      if (set.has(value)) {
+        return '[Circular]';
+      }
+      set.add(value);
+    } else if (typeof value === 'function') {
+      return value.toString();
+    } else if (value === Infinity) {
+      return 'Infinity';
```

Comment:
JSON.stringify returns null for this.  I presume it is intended that this differ from the spec?